packages/ffe-searchable-dropdown-react/less/ffe-multiselect-searchable-dropdown.less

```diff
@@ -0,0 +1,203 @@
+.ffe-searchable-dropdown-multiselect {
```

Comment:
Burde ikke vldig mye stilar kunna gjenbrukas mellom denne og searchable dropdown?

Reply:
Jeg prøvde.... Det var litt for mye ulikt dessverre